CHANGELOG.md

```diff
@@ -44,6 +44,7 @@
    * FIXED: Some interpolated points had invalid edge_index in trace_attributes response [#3646](https://github.com/valhalla/valhalla/pull/3670)
 
 * **Enhancement**
+   * CHANGED: Some refactoring of `CostMatrix` and remove duplicate implementation of `adjust_scores` [#3673](https://github.com/valhalla/valhalla/pull/3673)
```

Comment:
this goes at the bottom of the list. within each section we put the newer changes at the bottom